src/consensus/params.h

```diff
@@ -14,6 +14,20 @@ namespace Consensus {
  */
 struct Params {
     uint256 hashGenesisBlock;
+    /** Needs to evenly divide MAX_SUBSIDY to avoid rounding errors. */
+    int nSubsidySlowStartInterval;
+    /**
+     * Shift based on a linear ramp for slow start:
+     *
+     * MAX_SUBSIDY*(t_s/2 + t/r) = MAX_SUBSIDY*t_h  Coin balance
```

Comment:
Should that be `t_r` instead of `t/r`? The equation makes sense to me if it's `t_r`.